wordpress-mobile/WordPress-Android
# Merge develop into master post filters

Merges `develop` into `feature/master-post-filters`

To test:
- I don't think it's necessary to test it, but if you want to be sure.

1. Open Blog Posts
2. Make sure the action buttons have color
3. Update post and turn on airplane mode before the post gets uploaded
4. When the upload fails a **red** retry button should appear

cc @drspaceboo
